client/my-sites/upgrades/domain-management/domain-connect/domain-connect-authorize-description.jsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { translate } from 'i18n-calypso';
+
+class DomainConnectAuthorizeDescription extends Component {
+	placeholder = () => {
+		return (
+			<div className="domain-connect__is-placeholder">
+				<span></span>
+				<span></span>
+				<span></span>
+			</div>
+		);
+	}
+
+	getDescription = () => {
+		const { providerId } = this.props;
+
+		switch ( providerId ) {
+			case 'g-suite':
+				return translate( 'Howdy! It looks like you want to make your domain work with the ' +
+					'{{strong}}Google G Suite email service{{/strong}}.', {
```

Comment:
The product's name is just `G Suite` - `Google G Suite` sounds wrong 😀 Maybe something like `It looks like you want to make your domain work with Google's G Suite email service.` would be better? (so as to not loose whose service it is, but keep the product name "proper")